colabs/raytune/W&B + RayTune.ipynb

```diff
@@ -6,7 +6,8 @@
       "name": "W&B-RayTune.ipynb",
```

Comment:
<p>There's no need to bold <strong>W&amp;B</strong> at the beginning. <em>Don't forget to always use Ray/Tune</em>.</p><p> </p><p>The text is nice: it makes clear why you'd choose the <code>Logger</code> over the <code>mixin</code> and vice versa. I've got some edits, though.</p><p> </p><p>You open up with "You can either use..." but then end the sentence without saying what else you might use. "Either" should always be paired with "or".</p><p> </p><p>I suggest a rewrite, like so:</p><p> </p><blockquote>1.&nbsp;<code>WandbLogger</code> automatically logs metrics reported to Tune to W&amp;B along with the configuration of the experiment, using <code>tune</code>'s <code>logger</code> interface</blockquote><blockquote>2. The <code>@wandb_mixin</code> decorator gives you greater control over logging by letting you call <code>wandb.log</code> inside the decorated function, allowing you to log custom metrics, plots, and other outputs, like media.</blockquote><blockquote><br></blockquote><blockquote>These methods can be used together or independently. (<strong>no bolding</strong>)</blockquote><blockquote>The example below demonstrates how they can be used together.</blockquote><p>At this point, I would insert another markdown cell, containing the text below, then a code cell with the <code>import</code>s</p><blockquote>First, we import the relevant libraries, including some contents of a useful <code>examples</code>  module from <code>ray.tune</code>. </blockquote><p>then another markdown cell:</p><blockquote> Then, we define our training process, decorated with <code>@wandb_mixin</code>   so we can call <code>wandb.log</code>  to log our custom metric (here, the error rate; you might also log media, e.g. images from the training set).</blockquote><p>and then finally the example code.</p><p> </p><p>Avoid using "Wandb" unless you're referring to an object in code; in that case, it should be in monospaced/"code" style: <code>wandb</code> or <code>WandbOtherStuff</code>. If you're talking about the web app or the company, use "W&amp;B" or "Weights &amp; Biases".</p><p> </p><p>I would recommend that you don't include these examples here, since you're about to demonstrate them below.</p><p>While we're here, though, a few tips: make sure you visually align your code (e.g. <code>"Accuracy"</code> and <code>"Output"</code> should line up). If you're including an ellipsis in Python code (<code>...</code>) you need to make sure that any indentation-changing code is included (here, the <code>return</code>), otherwise it looks a bit weird. The whitespace in this example is also excessive: the lines above, between, and below the keys in the dictionary should be removed.</p><p> </p><p>The <code>WandbLogger</code> example looks weird here, but it's fine in the <a href="https://colab.research.google.com/github/AyushExel/examples/blob/colabs/raytune-torch/colabs/raytune/W%26B%20%2B%20RayTune.ipynb#scrollTo=o2AldspdQLlQ" target="_blank">Colab</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>